packages/playground/src/fonts.css

```
@@ -0,0 +1,32 @@
+  font-style: normal;
+  font-display: swap;
+}
+@font-face {
+  font-family: 'GeistMono';
+  src:
+    local('GeistMono'),
+    url('./assets/fonts/GeistMonoVariableVF.woff2') format('woff2-variations');
+  font-weight: 100 900;
+  font-style: normal;
+  font-display: swap;
+}
```

Stylelint (17.10.0)

[error] Expected empty line before at-rule (at-rule-empty-line-before)

(at-rule-empty-line-before)